### ISSUE TITLE
Special characters not supported in slim.

### ISSUE THREAD

**Issue by efauske** [unclebob/fitnesse#1008]
I'm having problems using special Norwegian characters, such as "æ,ø,å" in FitNesse using slim.
Here is an example:

![image](https://cloud.githubusercontent.com/assets/24959338/21719087/1419b1fc-d41c-11e6-9df3-622c856e605c.png)
![image](https://cloud.githubusercontent.com/assets/24959338/21719099/2649bbb0-d41c-11e6-8772-f325f7591fec.png)

When I run this test, "Test not run" appears:
![image](https://cloud.githubusercontent.com/assets/24959338/21719123/47cef228-d41c-11e6-9f41-3c5ed37f5d1d.png)

If i remove the 'æ' character, the test passes:
![image](https://cloud.githubusercontent.com/assets/24959338/21719142/63746bd4-d41c-11e6-837e-d172084ef1f1.png)

This occurred after I upgraded from v20150114 to v20161106, but I have tested it in v20160618 and it works fine there.

Can anyone help me solve this issue?

**Comment by efauske** [comment]
The test passed without issue in v20150114:

![image](https://cloud.githubusercontent.com/assets/24959338/21719383/cfe2c59e-d41d-11e6-90fc-d2d59506de23.png)

**Comment by keetron** [comment]
Could it be similar in any way to issue #995 ?
The solution for multi-byte chars is outlined there, it seems related to the version you mention.

**Comment by efauske** [comment]
Yes, you are right @keetron . Adding `!define slim.port {0}` solves the issue.

**Comment by amolenaar** [comment]
What OS are you using? Windows 7?

**Comment by efauske** [comment]
Yes @amolenaar , Windows 7.
